scripts/generate-bootstraps.sh

```diff
@@ -295,6 +322,11 @@ show_usage() {
 	echo "                             Multiple packages should be passed as"
 	echo "                             comma-separated list."
 	echo
+	echo " -c, --cache [DIR]           Enable caching. Downloaded package files will be"
```

Comment:
Ideally separate parameters should be used for enabling cache and passing custom cache dir in case other options need to be added in future, like `-c/--cache` and `--cache-dir`.